lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -472,7 +491,27 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
           inputSliceIndices.push_back(0);
           outputSliceIndices.push_back(0);
           assumedDynamicDimNotSplit = true;
-        } else {
+        } else if (inputShapeSlice[0] == 1 && outputShapeSlice[0] == kUnknownSize) {
+	  int64_t idx = 0;
+          while (idx < inputShapeSlice.size() && inputShapeSlice[idx] == 1) {
+            inputSliceIndices.push_back(idx++);
+	  }
+	  if (idx < inputShapeSlice.size() && inputShapeSlice[idx] == kUnknownSize) {
```

Comment:
We should also have a `checkDimEqualHelper` call in here, similar to the `else if (inputShapeSlice[0] == kUnknownSize)` case above